keyboards/synthlabs/065/info.json

```diff
@@ -0,0 +1,432 @@
+{
+  "keyboard_name": "065",
+  "manufacturer": "Synth Labs",
+  "url": "http://065.synthlabs.io",
+  "maintainer": "hongaaronc",
+  "processor": "RP2040",
+  "bootloader": "rp2040",
+  "diode_direction": "COL2ROW",
+  "debounce": 5,
```

Comment:
```suggestion
```

Reply:
Same as https://github.com/qmk/qmk_firmware/pull/21040#discussion_r1207831452. Applied in 
950abcd.